From 4c68ee8c490c0a8efe552f73c9597e914a040aca Mon Sep 17 00:00:00 2001
From: Christopher Degawa <ccom@randomderp.com>
Date: Sat, 23 Dec 2023 20:11:58 -0600
Subject: [PATCH] Initialize: Include array to fix compilation issue

Causes a compiler error with mingw-w64 g++ as it pulls the declaration
from tuple instead of std::array.

Signed-off-by: Christopher Degawa <ccom@randomderp.com>
---
 glslang/MachineIndependent/Initialize.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/glslang/MachineIndependent/Initialize.cpp b/glslang/MachineIndependent/Initialize.cpp
index 8632d6d6..39f5a243 100755
--- a/glslang/MachineIndependent/Initialize.cpp
+++ b/glslang/MachineIndependent/Initialize.cpp
@@ -51,6 +51,7 @@
 //                                                including identifying what extensions are needed if a version does not allow a symbol
 //
 
+#include <array>
 #include "Initialize.h"
 #include "span.h"
 
-- 
2.43.0

